cmake/developer_package/plugins/plugins.cmake

```diff
@@ -3,6 +3,7 @@
 #
 
 include(CMakeParseArguments)
+include(${CMAKE_CURRENT_LIST_DIR}/../clang_tidy/clang_tidy.cmake)
```

Comment:
I think you need to include it to developer package similar to clang-format, so all other components (not only inference plugins) can leverage that clang-tidy has been found

Reply:
Done